frontend/src/layout/Burger.tsx

```diff
@@ -16,28 +16,52 @@ export const BurgerMenu: React.FC<BurgerMenuProps> = ({ hide, items }) => (
             bottom: 0,
             left: 0,
             right: 0,
-            zIndex: 3017,
+            zIndex: 1000,
             backgroundColor: "#000000a0",
         }}
     >
-        <div css={{
+        <div tabIndex={-1} css={{
             position: "absolute",
             display: "flex",
             flexDirection: "column",
             justifyContent: "space-between",
             top: 0,
             right: 0,
-            backgroundColor: "var(--grey97)",
+            backgroundColor: "white",
             height: "100%",
             width: "clamp(260px, 75%, 450px)",
             overflowY: "auto",
             borderTop: "1px solid var(--grey80)",
-            gap: 16,
-            "& > *:first-child": {
-                borderBottom: "1px dashed var(--grey80)",
+            "> :nth-child(2)": { marginTop: 26 },
+            li: {
+                borderRadius: 4,
+                margin: 8,
+                borderBottom: "none",
+                "a, span": {
+                    padding: 16,
+                    borderRadius: 4,
+                },
             },
-            "& > *:last-child": {
-                borderTop: "1px dashed var(--grey80)",
+            nav: {
+                "> a": {
+                    margin: 8,
+                    padding: "12px 4px",
+                    borderRadius: 4,
+                    "svg:first-of-type": {
+                        display: "block",
+                        fontSize: 24,
+                    },
+                    "svg:last-of-type": { display: "none" },
```

Comment:
Interesting. Thinking about it... should maybe the curved arrow icon be used in the desktop nav as well? Honestly not sure. We can keep it as is :shrug: 

![image](https://user-images.githubusercontent.com/7419664/221609962-a8c8f489-70ba-4758-8a3a-5ccb366e0921.png)

Reply:
I think I would lean towards keeping the chevron thing here. Also not super sure, but somehow the arrow pointing up at the logo above it feels a little weird.